[Manufacturing]-[DOCS]Merge Get Started Sections (#3389)
open-edge-platform/edge-ai-suites: manufacturing-ai-suite/industrial-edge-insights-vision/docs/user-guide/get-started/deploy-multiple-instances-with-helm.md

   ------------------------------------------
   Processing instance: pcb1 from SAMPLE_APP: pcb-anomaly-detection
   ------------------------------------------

> 

Reviewer: ```suggestion
```
Why are we adding a blank line here inside the text code block? (The output for the other two instances doesn't have a blank line under the header)

   ```sh

   POD_NAME=$(kubectl get pods -n <INSTANCE_NAME> -o jsonpath='{.items[*].metadata.name}' | tr ' ' '\n' | grep deployment-dlstreamer-pipeline-server | head -n 1)
   
   # Replace "<VIDEOS_PATH>" and "<MODELS_PATH>" with the values for your sample application from the table that follows:

   kubectl cp resources/<VIDEOS_PATH>.avi $POD_NAME:/home/pipeline-server/resources/videos/ -c dlstreamer-pipeline-server -n <INSTANCE_NAME>

   kubectl cp resources/<MODELS_PATH>/models/* $POD_NAME:/home/pipeline-server/resources/models/ -c dlstreamer-pipeline-server -n <INSTANCE_NAME>
   ```
  
   | Application   | <VIDEOS_PATH> Value                    |
   | :----- | :--------------------------------------- |
   | Pallet Defect Detection  | pallet-defect-detection/videos/warehouse |
   | PCB Anomaly Detection   | pcb-anomaly-detection/videos/anomalib_pcb_test |
   
   | Application   | <MODELS_PATH> Value                    |
   | :----- | :--------------------------------------- |
   | Pallet Defect Detection  | pallet-defect-detection |
   | PCB Anomaly Detection   | pcb-anomaly-detection |

4. Install the package `boto3` in your python environment if not installed.

>    Intel recommends creating a virtual environment and install it there. You can run the following commands to add the necessary dependencies as well as create and activate the environment.

Reviewer: ```suggestion
   Intel recommends creating a virtual environment and installing it there. You can run the following commands to add the necessary dependencies as well as create and activate the environment.
```

Reviewer 2: I'd also be cautious about the statement as such. Maybe we could go with "you should use a VM" instead of "it is recommended / someone recommends"

                  "type": "uri"
               },
               "destination": {
               "frame": {
                  "type": "webrtc",
                  "peer-id": "anomaly"
               }
               },
               "parameters": {
                  "classification-properties": {
                     "model": "/home/pipeline-server/resources/models/pcb-anomaly-detection/deployment/Anomaly classification/model/model.xml",
                     "device": "CPU"
                  }
               }
         }
      }
   ]
   ```

5. Start the pipeline with the above payload.

   Replace "<APP>" with your sample application from the table that follows:

   ```sh
   ./sample_start.sh helm -i <INSTANCE_NAME> -p <APP>_mlops
   ```
   Note the instance-id.
   
   | Application   | <APP> Value                    |
   | :----- | :--------------------------------------- |
   | Pallet Defect Detection  | pallet_defect_detection |
   | PCB Anomaly Detection   | pcb_anomaly_detection |

6. Download and prepare the model. Replace <MODEL_PATH> with the desired value from the table that follows: 

>    > **Note:** For sake of simplicity, Intel assumes that the new model has already been downloaded by the Model Download microservice. The following curl command is only a simulation that downloads the model. In production, however, they will be downloaded by the Model Download service.

Reviewer: ```suggestion
   > **Note:** For sake of simplicity, we assume that the new model has already been downloaded by the Model Download microservice. The following curl command is only a simulation that downloads the model. In production, however, they will be downloaded by the Model Download service.
```
I think "we assume" here is, uh, less imposing and safer (if we won't use passive voice).

Reviewer 2: I agree, using the company name in this kind of docs may be taken as improper. It may sound as if it is the official stance of the entire company.

Author: Changed to "these instructions assume...".
